### ISSUE TITLE
Eroare cu repayment fee activat

### ISSUE THREAD

**Issue by raulw** [sameday-courier/woocommerce-plugin#117]
Eroare la "add to cart" cand este activata noua functie din 1.4.3 - "repayment fee". 

Nu stiu inca care este legatura (nu am "investigat"), doar am testat rapid pe 2 site-uri diferite, acelasi comportament,  in momentul in care este setat "repayment fee", da eroare "_wc-ajax=add_to_cart:1_" iar in log:

 _Class "SamedayCourier_Shipping_Method" not found in /public_html/wp-content/plugins/samedaycourier-shipping/samedaycourier-shipping.php:624_

**Comment by Soptareanu** [comment]
> Eroare la "add to cart" cand este activata noua functie din 1.4.3 - "repayment fee".
> 
> Nu stiu inca care este legatura (nu am "investigat"), doar am testat rapid pe 2 site-uri diferite, acelasi comportament, in momentul in care este setat "repayment fee", da eroare "_wc-ajax=add_to_cart:1_" iar in log:
> 
> _Class "SamedayCourier_Shipping_Method" not found in /public_html/wp-content/plugins/samedaycourier-shipping/samedaycourier-shipping.php:624_

Buna ziua!
         Am facut un fix. Asteptam un feedback de la dvs. sa ne confirmati daca a fost remediata situatia sau in continuare intampinati disfunctionalitati in site.

**Comment by raulw** [comment]
Salut! 
Da, am observat si am si facut update. Totul este okei acum. 
Offtopic: legat tot de acest "repayment fee", am cumva posibilitatea sa specific daca include sau nu tva? 

Merci mult!

**Comment by Soptareanu** [comment]
Nu, valoarea este fixa. Coincide cu suma setata in campul respectiv, din setarile pluginului. Tva-ul il aplica woocommerce la sfârșit pe suma totala din cos.

Trimis din Yahoo Mail pe Android 
 
  Pe mar., ian. 10, 2023 la 14:27, Raul ***@***.***> a scris:   


Salut!
Da, am observat si am si facut update. Totul este okei acum.
Offtopic: legat tot de acest "repayment fee", am cumva posibilitatea sa specific daca include sau nu tva?

Merci mult!
